frontend/public/components/app-contents.tsx

```diff
@@ -449,7 +449,7 @@ const AppContents: React.FC<{}> = () => {
             />
 
             <LazyRoute
-              path={`/k8s/ns/:ns/:plural/:name/${VolumeSnapshotModel.plural}/~new/form`}
+              path={`/k8s/ns/:ns/${VolumeSnapshotModel.plural}/pvc/:pvcName/~new/form`}
```

Comment:
I would recommend using a query parameter instead like

```
/k8s/ns/:ns/${VolumeSnapshotModel.plural}/~new/form?pvc=my-pvc
```

Then we don't need a separate route. This follows the existing URL pattern we have better as well.

Reply:
Sure, makes sense. I guess the query param doesn't have to be declare in the route definition.

Reply:
done